### DISCUSSION TITLE
能否在后台新增关闭下载设置

### DISCUSSION THREAD

**Discussion by qadan72** [AlistGo/alist#866] [Ideas]
如果访客下载流量过多，账号可能会被阿里云封，下个版本能增加下吗

**Comment by xhofe** [comment]
。。。那你直接删除账号啊，不能下载挂着干嘛，看目录结构？

**Reply by qadan72** [reply]
> ……删除账号啊，不能下载挂着你那……看目录结构直接？

网盘主要内容是视频，视频可以在线看，把后端，域名/d，这个下载接口关掉，或者在前端隐藏右键，操作，下载，老版本的alist前端我可以改，现在是tsx语法不熟悉，操作不了

**Reply by xhofe** [reply]
视频还不是能拿到下载链接。无意义

**Reply by qadan72** [reply]
> 视频还不是能拿到下载链接。无意义

那我还是自己改吧

**Reply by yyy33** [reply]
二

> 视频还不是能拿到下载链接。无意义

请教下作者关于阿里云流量异常封号的事,
在alist server 关闭所有中转设置的情况下,client(client和alist server不是同一个IP)向alist server请求文件,然后alist server向阿里云请求文件,收到阿里云的回复后再发送302重定向至client, client 根据302的信息再向阿里云请求文件,请问client此时向阿里云发送的请求里有账号相关的信息吗,
如果有的话阿里云是不是就是根据你两次发送请求源地址不是同一个IP地址,然后判断你账号流量异常,从而封你的账号.

**Reply by xhofe** [reply]
> 二
> 
> > 视频还不是能拿到下载链接。无意义
> 
> 请教下作者关于阿里云流量异常封号的事, 在alist server 关闭所有中转设置的情况下,client(client和alist server不是同一个IP)向alist server请求文件,然后alist server向阿里云请求文件,收到阿里云的回复后再发送302重定向至client, client 根据302的信息再向阿里云请求文件,请问client此时向阿里云发送的请求里有账号相关的信息吗, 如果有的话阿里云是不是就是根据你两次发送请求源地址不是同一个IP地址,然后判断你账号流量异常,从而封你的账号.

这个你得问阿里，目测应该是有账号信息的。

**Reply by yyy33** [reply]
> > 二
> > > 视频还不是能拿到下载链接。无意义
> > 
> > 
> > 请教下作者关于阿里云流量异常封号的事, 在alist server 关闭所有中转设置的情况下,client(client和alist server不是同一个IP)向alist server请求文件,然后alist server向阿里云请求文件,收到阿里云的回复后再发送302重定向至client, client 根据302的信息再向阿里云请求文件,请问client此时向阿里云发送的请求里有账号相关的信息吗, 如果有的话阿里云是不是就是根据你两次发送请求源地址不是同一个IP地址,然后判断你账号流量异常,从而封你的账号.
> 
> 这个你得问阿里，目测应该是有账号信息的。

看来只能使用中转了,保险了一点

**Comment by huyangnl** [comment]
关闭下载，关闭预览这些功能都需要有开关。请考虑增加。
